src/edge/edgeManager.ts

```diff
@@ -778,7 +796,7 @@ export class EdgeManager {
         if (!templatePick) {
             throw new UserCancelledError();
         }
-        TelemetryClient.sendEvent( `${Constants.addModuleEvent}.selectModuleTemplate`, {
+        TelemetryClient.sendEvent(`${Constants.addModuleEvent}.selectModuleTemplate`, {

```

Comment:
for this space, we could add config "space-within-parens": true into tslint.json